### PR TITLE
Update rasa-pro.mdx

### PR DESCRIPTION
**Pull request by andreaklmn**
Links for PII and Markers under Read More have to be added!

**Proposed changes**:
- ...

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: docs/docs/rasa-pro.mdx [hunk 1]
```diff
@@ -44,6 +44,18 @@ Resolve performance issues faster and identify bottlenecks in message handling a
 
 [Read more here](./monitoring/tracing.mdx).
 
+### PII Handling
+
+Anonymize PII (Personal Identifiable Information) in logs and events streamed via the Kafka event broker. 
+
+[Read more here]
```
Comment (m-vdb) [inline docs/docs/rasa-pro.mdx:51]: ah basically we can't merge it until we have the links to the docs from Anca / Shailendra

Comment (andreaklmn) [reply docs/docs/rasa-pro.mdx:51]: now I don't know how to add it in but I fished out the links: (./pii-management.mdx)

File: docs/docs/rasa-pro.mdx [hunk 2]
```diff
@@ -44,6 +44,18 @@ Resolve performance issues faster and identify bottlenecks in message handling a
...
+
+### Real-Time Markers
+
+Mark points of interest in conversations to support the targeted analysis of user journeys real time.
+
+[Read more here]
```
Comment (andreaklmn) [inline docs/docs/rasa-pro.mdx:57]: (./monitoring/analytics/realtime-markers.mdx)

File: docs/docs/rasa-pro.mdx (hunk 1 above)
Comment (andreaklmn) [inline docs/docs/rasa-pro.mdx:51]:
```suggestion
[Read more here] (./pii-management.mdx)
```

File: docs/docs/rasa-pro.mdx (hunk 2 above)
Comment (andreaklmn) [inline docs/docs/rasa-pro.mdx:57]:
```suggestion
[Read more here] (./monitoring/analytics/realtime-markers.mdx)
```

Comment (m-vdb) [reply docs/docs/rasa-pro.mdx:57]: https://github.com/RasaHQ/rasa/pull/12527

**Review by m-vdb — APPROVED** [review]

**Pull request merged**